generic/tests/netstress_kill_guest.py

```diff
@@ -71,19 +127,11 @@ def netload_kill_problem(test, session_serial):
                                    bg_stress_test), logging.info)
             stress_thread = None
             wait_time = int(params.get("wait_bg_time", 60))
-            bg_stress_run_flag = params.get("bg_stress_run_flag")
             vm_wait_time = int(params.get("wait_before_kill_vm"))
-            env[bg_stress_run_flag] = False
-            stress_thread = utils_misc.InterruptedThread(
-                utils_test.run_virt_sub_test, (test, params, env),
-                {"sub_type": bg_stress_test})
```

Comment:
`bg_stress_test = params.get("background_stress_test", netperf_stress')`
In your patch, the background test will always be set to "netperf", so `background_stress_test` will become meaningless in cfg.

Reply:
Thanks for your reminder, I have changed it.